### ISSUE TITLE
01-basics - Logistic regression example 

### ISSUE THREAD

**Issue by int8** [yunjey/pytorch-tutorial#54]
hi there, 

First of all, This is really good tutorial, thank you for your work! 

I am having some troubles understanding Logistic Regression example [here](https://github.com/yunjey/pytorch-tutorial/blob/master/tutorials/01-basics/logistic_regression/main.py). What bothers me is lack of clear activation function at the very end of the graph. From what I see I conclude this is in fact softmax classifier rather than logistic regression. Am I missing smth?

**Comment by yunjey** [comment]
You can see [here](https://github.com/yunjey/pytorch-tutorial/issues/17). `nn.CrossEntropy` computes softmax internally.This is same as when we use softmax with [nn.NLLLoss](http://pytorch.org/docs/master/nn.html#nllloss).

**Comment by int8** [comment]
sure, I understand softmax exists there implicitly via error function formulation - what I do not see is [logistic](https://en.wikipedia.org/wiki/Logistic_function) part of logistic regresstion

**Comment by yunjey** [comment]
logistic sigmoid is just special case of softmax (when the number of class is 2). In multi-class classification task, we use softmax instead of sigmoid. [This](https://stats.stackexchange.com/questions/233658/softmax-vs-sigmoid-function-in-logistic-classifier) will help you.

**Comment by int8** [comment]
thanks a lot - helped indeed, take care
